src/index.html

```diff
@@ -9,8 +9,39 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700;900&family=Roboto:wght@300;400&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+      class="search__block"
+      action="/"
+      method="post"
+    >
+
+      <input
+        data-qa="big"
+        class="input__search search--big"
+        type="search"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+    </form>
+
+    <form
+      class="search__block"
+      action="/"
+      method="post"
+      >
+
+      <input
+        data-qa="small"
+        class="input__search search--small"
+        type="search"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+    </form>
+
```

Comment:
```suggestion
    <form
      class="search__block"
      action="/"
      method="post"
    >
      <input
        data-qa="big"
        class="input__search search--big"
        type="search"
        placeholder="Try &quot;Los Angeles&quot;"
      >
    </form>

    <form
      class="search__block"
      action="/"
      method="post"
    >

      <input
        data-qa="small"
        class="input__search search--small"
        type="search"
        placeholder="Try &quot;Los Angeles&quot;"
      >
    </form>
```
don't add spaces between parent and child elements, also closing bracket should be on the same level as opening one

Reply:
5'th point in checklist